.env.test

```diff
@@ -1,2 +1,3 @@
 VALIDATOR_URL=https://example.com/validatorapi
+FHIR_RESOURCE_VALIDATOR_URL=http://localhost/hl7validatorapi
```

Comment:
You should make this something fake like the other one to ensure that the tests are never hitting a real service.